### PR TITLE
 fix observable default assumption in Example 4

### PR DESCRIPTION
**Pull request by sebastiankb**
solve issue #1293

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/1307.html" title="Last updated on Dec 1, 2021, 2:57 PM UTC (6620583)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1307/ea62552...6620583.html" title="Last updated on Dec 1, 2021, 2:57 PM UTC (6620583)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5321,6 +5322,15 @@ <h3>Protocol Bindings</h3>
...
     </ul>
+
+    <p class="note">
+        Optimally, the protocols used are listed as a scheme in the IANA registry [[?IANA-URI-SCHEMES]]). 
+        This guarantees a unique <a>Protocol Binding</a> assignment. In case the desired protocol is not yet 
+        registered with IANA, it is recommended to follow the schema rules of the protocol specifications, if available.
+        Otherwise, established implementations of the protocols specify a defacto scheme that should also be followed for 
+        TD definitions. If an IANA entry is not available, the <a>Protocol Binding</a> document will 
+        provide a recommended schema value to allow unique protocol identification in the context of WoT.
+    </p>
```
Comment (danielpeintner) [inline index.html:5333]: I am not 100% sure why this note was added to this PR.

Having said that, I seldomly read "Optimally" ... more often "Ideally" .. but this should be judged by a native speaker. 

Moreover, I believe "defacto" should become "de facto" or "de-facto"

**Review by danielpeintner — APPROVED** [review]
The actual fix for https://github.com/w3c/wot-thing-description/issues/1293 looks good.

I am a bit unsure about the unrelated note that was added.

File: index.html (hunk 1 above)
Comment (sebastiankb) [reply index.html:5333]: strange, this is showing up here. This is related to PR #1309 

I will remove it here again

**Comment by sebastiankb** [comment]
from today's TD call, group decided to merge

**Pull request merged**